footprints/templates/main/gallery_view.html

```diff
@@ -0,0 +1,20 @@
+<!--Gallery View for search.html-->
+
+<div class="gallery-container">
+    {% for result in object_list %}
+        {% with footprint=result.object %}
+            <div class="panel paned-default gallery-item">
+                <div class="panel-body">
+                    <a href="/footprint/{{footprint.id}}">
+                        <img src="{{footprint.digital_object.url}}" alt="{{footprint.title}}">
```

Comment:
`digital_object` is a ManyToMany field. We'll need to iterate this set to display all the photos associated with a footprint. This change can be made in the next PR.